sdk/storage/storage-blob/test/blobclientpollers.spec.ts

```diff
@@ -94,47 +97,57 @@ describe("BlobClient beginCopyFromURL Poller", () => {
     assert.deepStrictEqual(properties2.copySource, blobClient.url);
   });
 
-  // These tests depend on the service not returning 'success' as soon as
-  // the copy is initiated. Since this can't be guaranteed in the live tests,
-  // these tests will only run with the unit tests with pre-recorded service responses.
-  if (env.TEST_MODE === "playback") {
-    it("supports cancellation of the copy", async () => {
-      const newBlobClient = destinationContainerClient.getBlobClient(
-        recorder.getUniqueName("copiedblob")
-      );
-      const poller = await newBlobClient.beginCopyFromURL(
-        "https://raw.githubusercontent.com/Azure/azure-sdk-for-js/master/README.md",
-        testPollerProperties
-      );
-      await poller.cancelOperation();
-      try {
-        await poller.pollUntilDone();
-        throw new Error("Test failure");
-      } catch (err) {
-        assert.equal(err.name, "PollerCancelledError");
-      }
-    });
-
-    it("supports updating on progress events", async () => {
-      const newBlobClient = destinationContainerClient.getBlobClient(
-        recorder.getUniqueName("copiedblob")
-      );
-      let onProgressCalled = false;
-      const poller = await newBlobClient.beginCopyFromURL(
-        "https://raw.githubusercontent.com/Azure/azure-sdk-for-js/master/README.md",
-        {
-          onProgress(_) {
-            onProgressCalled = true;
-          },
-          ...testPollerProperties
-        }
-      );
+  it("supports cancellation of the copy", async function() {
+    if (!(isRecordMode() || isPlaybackMode())) {
+      // Depends on the service not returning 'success' as soon as
+      // the copy is initiated. Since this can't be guaranteed in the live tests,
+      // these tests will only run with the unit tests with pre-recorded service responses.
+      this.skip();
+    }
```

Comment:
**Old** (- Hides the specific test if we run the integration tests)
```ts
// Reason
if (env.TEST_MODE === "playback") {
  it("supports cancellation of the copy", async () => {  /** test code */   });
}
```

**New** (- Shows that the test is being skipped during the integration test runs and will also run during the record mode for future playback)
```typescript
it("supports cancellation of the copy", async function() {
    if (!(isRecordMode() || isPlaybackMode())) {
      // Reason
      this.skip();
    }
});
```


/cc - @chradek @sadasant